File: Frontend/DisasterDispatchFrontend/src/app/Models/DTOS/CustomOperation/custom-operation-dto.ts
import { Guid } from "guid-typescript";
import { AppUser } from "../../app-user";
import { DisasterOperation } from "../../disaster-operation";
import { EmergencyReport } from "../../emergency-report";

export class CustomOperationDto {
    public id: Guid;
    public emergencyReportId: Guid;
    public emergencyReport: EmergencyReport | null;
    public disasterOperationId: Guid;
    public disasterOperation: DisasterOperation | null;
    public workers: AppUser[];
    public status: string;
}

File: Backend/DisasterDispatch.Core/Dtos/CustomOperationDtos/CustomOperationDto.cs
using DisasterDispatch.Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Dtos.CustomOperationDtos
{
    public class CustomOperationDto
    {
        public Guid Id { get; set; }
        public Guid EmergencyReportId { get; set; }
        public EmergencyReport EmergencyReport{ get; set; }
        public Guid DisasterOperationId { get; set; }       
        public DisasterOperation DisasterOperation{ get; set; }
        public ICollection<AppUser> Workers { get; set; }
        public string Status { get; set; }
    }
}
